sia/miner/info.go

```diff
@@ -29,7 +29,7 @@ type MinerStatus struct {
 // Address is the current address that is receiving block payouts.
 func (m *Miner) Info() ([]byte, error) {
 	m.mu.RLock()
-	defer m.mu.RLock()
+	defer m.mu.RUnlock()
```

Comment:
we spent an hour and then we discovered this was the cause of all the pain.

\>.< we should have found this so much faster.